sdk/storage/storage-file-share/README.md

```diff
@@ -104,10 +104,6 @@ The JS bundled file is compatible with [UMD](https://github.com/umdjs/umd) stand
 
 - `azfile`
 
-#### Download
-
-Download latest released JS bundles from links in the [GitHub release page](https://github.com/Azure/azure-storage-js/releases).
```

Comment:
The same `#### Download` section is present in storage-blob and storage-queue packages too.

Links -
https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/storage/storage-blob/README.md#download
https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/storage/storage-queue/README.md#download

Would you mind removing them as well? 🙂